edx_exams/apps/router/middleware.py

```diff
@@ -0,0 +1,35 @@
+"""
+Middleware that checks if in incoming request should be
+routed to the legacy proctoring service (edx-proctoring)
+"""
+from django.utils.deprecation import MiddlewareMixin
+
+from edx_exams.apps.api.v1.views import CourseExamsView
+from edx_exams.apps.core.models import CourseExamConfiguration
+from edx_exams.apps.router.views import CourseExamsLegacyView
+
+LEGACY_VIEW_MAP = {
```

Comment:
Question: Is this a map because we anticipate multiple views having legacy views to be mapped to?

Reply:
yup 👍